### PR TITLE
WIP: Praise summary page for period/giver 

### LINKED ISSUE #590: Praise summary page for period/giver
- You reach this page by clicking on a giver in the analytics page under the headlines "Givers by score" and "Top givers by number".
- URL: `/periods/[perioId]/giver/[userAccountId]`

Page contents, similar to giver page:

image

Clarify header message on both giver and receiver page.

Giver page:
`Total score, praise given: xxx`

Receiver page:
`Total score, praise received: xxx`

### PR DESCRIPTION
**Pull request by nebs-dev**
Resolves #590 

- Summary page for the period giver
- List of the givers in period detail page

### CODE REVIEW

File: packages/frontend/src/pages/Periods/GiverSummary/GiverSummaryPage.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import { GiverSummaryTable } from './components/GiverSummaryTable';
+
+const getGiver = (
+  periodDetails: PeriodDetailsDto,
+  giverId: string
+): PeriodDetailsGiverReceiverDto | undefined => {
+  return periodDetails.givers?.find((r) => r._id === giverId);
+};
+
+const PeriodReceiverMessage = (): JSX.Element | null => {
```
Comment (kristoferlund) [inline packages/frontend/src/pages/Periods/GiverSummary/GiverSummaryPage.tsx:23]: Rename to PeriodGiverMessage

File: packages/frontend/src/pages/Periods/Details/PeriodDetailsPage.tsx [hunk 2]
```diff
@@ -109,6 +110,13 @@ export const PeriodDetailsPage = (): JSX.Element | null => {
                 replace
                 rounded
               />
+              <NavItem
+                to={`${url}/givers`}
+                description="Givers"
+                icon={faHandsHoldingCircle}
```
Comment (kristoferlund) [inline packages/frontend/src/pages/Periods/Details/PeriodDetailsPage.tsx:116]: I suggest we use 'hand-holding-heart' for giver and 'hands-holding-heart' for receiver.

**Review by kristoferlund — CHANGES_REQUESTED** [review]
Looks great! I proposed two small changes only.

File: packages/frontend/src/pages/Periods/Details/PeriodDetailsPage.tsx (hunk 2 above)
Comment (nebs-dev) [reply packages/frontend/src/pages/Periods/Details/PeriodDetailsPage.tsx:116]: @kristoferlund hm, looks like `hands-holding-heart` is a pro icon

**Pull request merged**